internal/evaluation_target/application_snapshot_image/application_snapshot_image.go

```diff
@@ -59,21 +59,23 @@ type ApplicationSnapshotImage struct {
 	Evaluators       []evaluator.Evaluator
 	files            map[string]json.RawMessage
 	component        app.SnapshotComponent
+	snap             app.SnapshotSpec
```

Comment:
Nit, I'd use the full name
```suggestion
	snapshot         app.SnapshotSpec
```

Reply:
Done